packages/components/atoms/f-form-field/src/components/FormField.vue

```diff
@@ -282,17 +291,31 @@ export default {
 $form-input-textColour                    : $color-content-default;
 $form-input-textColour--disabled          : $color-content-disabled;
 $form-input-bg--disabled                  : $color-disabled-01;
+
 $form-input-borderRadius                  : $border-radius;
 $form-input-borderWidth                   : 1px;
-$form-input-borderColour                  : $color-border-strong;
+$form-input-borderColour                  : $color-border-default;
 $form-input-borderColour--focus           : $color-grey-50;
 $form-input-borderColour--invalid         : $color-support-error;
 $form-input-borderColour--disabled        : $color-disabled-01;
-$form-input-height                        : 46px; // height is 46px + 1px border = 48px
-$form-input-padding                       : spacing(x1.5) spacing(x2);
+
+$form-input-height-small                  : spacing(x5);
+$form-input-padding-small                 : spacing(x1) spacing(x2);
+
+$form-input-height-medium                 : spacing(x6);
```

Comment:
One thing worth checking with the height in the browser is if the height is 48px with the border included (and the same for small/large).

In the designs, the 48px includes the border inside that height, but the CSS box-model includes the border outside of the height, so you may end up with a form-field that's 50px height (if that makes sense).

We had a similar thing with buttons I think – would just mean you need to do factor in the border width when setting the height (so this could then be `spacing(x6) - ($form-input-borderWidth * 2)` for instance.

That would be why `$form-input-height` is set to `46px` in the old code though (see the comment `// height is 46px + 1px border = 48px`), so worth double checking in devtools and just updating in each size.